File: programmers/java/lev0/숨어있는 숫자의 덧셈 (1).java
class Solution {
    public int solution(String my_string) {
        int answer = 0;
        for(String s : my_string.split("")){
            try{
                answer += Integer.parseInt(s);
            }catch(java.lang.NumberFormatException e){} 
        }
        return answer;
    }
}

// stream을 이용해서 구하기
// 
class Solution {
    public int solution(String myString) {
        return myString.chars().mapToObj(i -> (char) i).filter(Character::isDigit).map(String::valueOf).mapToInt(Integer::valueOf).sum();
    }
}

File: programmers/js/lev0/숨어있는 숫자의 덧셈 (1).js
function solution(my_string) {
  return my_string
    .split("")
    .reduce((acc, el) => acc + (!isNaN(parseInt(el)) ? parseInt(el) : 0), 0);
  return [...my_string].reduce(
    (acc, el) => acc + (!isNaN(parseInt(el)) ? parseInt(el) : 0),
    0
  );
}
